src/app/containers/Timestamp/helpers/testHelpers.js

```diff
@@ -15,4 +18,4 @@ const timestampGenerator = timeDifference => {
   return timestamp;
 };
 
-export default timestampGenerator;
+export const isBritishSummerTime = timestamp => moment(timestamp).isDST();
```

Comment:
You might need to specify a region here. If this runs in the US, its idea of when DST is will be different

Reply:
I checked this and it seems [from the docs as if Moment's implementation of `isDST` is timezone/location aware](http://momentjs.com/docs/#/query/is-daylight-saving-time/)

Reply:
Looks like we need something like 
```
moment.tz(timestamp, 'Europe/London').isDST()
```